addons/cookoff/XEH_postInit.sqf

```diff
@@ -73,8 +73,11 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 
 // secondary explosions
 ["AllVehicles", "killed", {
-    params ["_vehicle"];
-    if (_vehicle getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
+    params ["_vehicle", "", "", "_useEffects"];
+    if (
+        _vehicle getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)] &&
+        _useEffects
```

Comment:
Using the shorter variable first would be a bit more readable imho `_useEffects &&`.

Reply:
Could also combine the `if` `exitWith` below into this as well.